data/en/test_unit_patterns.json

```diff
@@ -15,6 +15,9 @@
     }, {
         "name": "ru_name",
         "validator": "string"
+    }, {
+        "name": "language_code",
```

Comment:
Does this need to be added to all of them? Seems odd to only add it to the one schema

Reply:
Not really, only ones that need to support multiple languages

Reply:
Oki dokes, just checking